src/app/components/formElements/CheckBoxItem/CheckBoxItem.tsx

```diff
@@ -1,59 +1,61 @@
-import React, { HTMLAttributes, MouseEvent } from 'react';
+import React, { ButtonHTMLAttributes, MouseEvent } from 'react';
 import styles from './checkBoxItem.module.css';
 import classNames from 'classnames';
 import CheckIcon from '../../images/Check';
 
-interface PropType extends Omit<HTMLAttributes<HTMLButtonElement>, 'onChange'> {
+interface PropType
+  extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange' | 'type'> {
   label: string;
   selected: boolean;
   onChange: (selected: boolean, e: MouseEvent<HTMLButtonElement>) => void;
+  disabled?: boolean;
   type?: 'checkbox' | 'radio' | 'pill';
 }
 
 const CheckBoxItem: React.FC<PropType> = ({
   label,
   selected,
   type = 'checkbox',
+  disabled,
   onChange,
+  onClick: _onClick,
   ...rest
 }) => {
   const isTypeRadio = type === 'radio';
+  const isypeCheckbox = type === 'checkbox';
```

Comment:
Hi Salman, just a minor typo here. other than that, everything looks good!

Reply:
@mikemike27 Good catch. Can you please fix it and push? Thanks!